bin/src/prove.rs

```diff
@@ -44,6 +47,13 @@ impl Args {
         return true;
     }
 
+    fn does_create_verifier_sol(&self) -> bool {
+        match &self.sol {
+            Some(sol) => sol.clone(),
+            _ => false,
+        }
+    }
```

Comment:
If the type of `self.sol` is changed to `bool`, then this function seems not needed.